File: ruby/problem_5.rb
def test_num(test_number)
    test_set = [11, 12, 13, 14, 15, 16, 17, 18, 19, 20]
        for num in test_set
            if test_number % num != 0
                return false
            end
        end
    return true
end

for x in (2520..10000000000).step(2520) do
    if test_num(x)
        puts("Problem 5: " + String(x))
        break
    end
end

File: python/problem_5.py
def test_num(test):
    test_set = [11, 12, 13, 14, 15, 16, 17, 18, 19, 20]
    for num in test_set:
        if test % num != 0:
            return False
    return True

for x in range(2520, 10000000000, 2520):
    if test_num(x):
        print("Answer 5:", x)
        break
